index.html

```diff
@@ -1,313 +1,351 @@
-                    name="email"
-                    placeholder="Email address"
-                    required
-                    type="email"
-                  />
-                </li>
-                <li>
-                  <textarea
-                    class="inputTextarea"
-                    name="opinion"
-                    id="opinion"
-                    cols="50"
-                    rows="10"
-                    placeholder="Enter text here"
-                    maxlength="500"
-                    required
-                    type="text"
-                  ></textarea>
-                </li>
-                <li>
-                  <button type="submit" class="touch-button">Get In Touch</button>
-                </li>
-              </ul>
-            </form>
-          </section>
-          <section class="footer">
-            <footer>
-              <ul class="Social-Media">
-                <li>
-                  <a href="https://github.com/ibtisam34">
-                    <img src="images/github.png" alt="-GitHub" />
-                  </a>
-                </li>
-                <li>
-                  <a href="https://www.linkedin.com/in/riyangwl2">
-                    <img src="images/linkedn.png" alt="Linkedin" />
-                  </a>
-                </li>
-                <li>
-                  <a href="#">
-                    <img src="images/angelist.png" alt="Angelist" />
-                  </a>
-                </li>
-                <li>
-                  <a href="https://twitter.com/Queenjin12">
-                    <img src="images/twitter.png" alt="Twitter" />
-                  </a>
-                </li>
-                <li>
-                  <a href="#">
-                    <img src="images/medium.png" alt="Medium" />
-                  </a>
-                </li>
-              </ul>
-              <nav class="img">
-                <img src="Shape.png" alt="straight line">
-              </nav>
-            </footer>
-          </section>
+      <section class="touch" id="touch">
+        <h1>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h1>
+        <form
+          action="https://formspree.io/f/xjvzrlvl"
+          class="touchForm"
+          method="post"
+        >
+          <input
+            type="text"
+            placeholder="Full name"
+            maxlength="30"
+            name="name"
+            required
+          />
+          <input
+            type="email"
+            placeholder="Email address"
+            name="email"
+            required
+          />
+          <textarea
+            name=""
+            id=""
+            cols="30"
+            rows="10"
+            type="text"
+            placeholder="Enter text here"
+            maxlength="500"
+            required
+          ></textarea>
+          <button class="button" type="submit">Get in touch</button>
```

Comment:
- [ ] to match Figma design this button should be aligned to the left , kindly fix that

![image](https://user-images.githubusercontent.com/89282221/194702135-4a9ed3f1-ee48-4a1c-951d-8f68b8ffd672.png)